diagrams/aws/devtools.py

```diff
@@ -20,6 +20,10 @@ class Cloud9(_Devtools):
     _icon = "cloud9.png"
 
 
+class Codeartifact(_Devtools):
+    _icon = "codeartifact.png"
+
+
```

Comment:
Same here for the name the aws console page for code artifact is display like this: 

**AWS CodeArtifact**

Reply:
hmm i think currently we currently use this kind of naming for all devtool services so I don't this one should be change.
![image](https://github.com/user-attachments/assets/af0e3b72-4069-4d5f-a426-5f3ef5ea6300)